### PR TITLE
Update creating an image section

### PR DESCRIPTION
**Pull request by K-Meech**
This PR updates the 'Creating an image' section of the 'What is an image?' episode. This incorporates Nic Cade's comments.

@davecash75 - could you take a quick look and merge if you're happy?

### CODE REVIEW

File: episodes/what-is-an-image.md [hunk 1]
```diff
@@ -160,12 +160,15 @@ detection
...
 e.g. a CCD camera
 
-To briefly summarise, light is emitted from the sample (e.g. from a fluorescent 
-marker), then gathered and focused with the objective lens, before hitting the 
-detector. The detector is divided into many small, light sensitive areas - these 
-are physical pixels, that will correspond to the pixels in the final image. When 
-light hits one of these areas it is converted to one or more electrons, with 
-more light resulting in more electrons and a higher final value for that pixel.
+To briefly summarise for a fluorescence microscopy image:   
+an excitation light source (e.g. a laser) illuminates the sample, a fluorescent
```
Comment (davecash75) [inline episodes/what-is-an-image.md:164]: Maybe a slight tweak of "an excitation light source (e.g. a laser illuminates the sample, and this light is absorbed by (or excites a?) fluorescent label, causing it to emit light..."

**Review by davecash75 — COMMENTED** [review]
Only a minor suggestion. I'll leave it up to you whether you think it works or not.

**Review by davecash75 — APPROVED** [review]
Sorry, should have marked this approved, as I don't want to block the merge over my comment.

File: episodes/what-is-an-image.md (hunk 1 above)
Comment (K-Meech) [reply episodes/what-is-an-image.md:164]: I'll add this small change and then merge - thanks!

**Pull request merged**